src/containers/BroadcastTransactionCoin/HederaForm.tsx

```diff
@@ -0,0 +1,71 @@
+import { Form, FormikHelpers, FormikProvider, useFormik } from 'formik';
+import { Link } from 'react-router-dom';
+import * as Yup from 'yup';
+import { Button, FormikTextfield } from '~/components';
+
+const validationSchema = Yup.object({
+  serializedSignedTx: Yup.string().required(),
+  startTime: Yup.number()
+    .optional()
+    .min(Date.now() / 1000, 'Start time must be in the future'),
+}).required();
+
+export type HederaFormProps = {
+  onSubmit: (
+    values: HederaFormValues,
+    formikHelpers: FormikHelpers<HederaFormValues>
+  ) => void | Promise<void>;
+};
+
+type HederaFormValues = Yup.Asserts<typeof validationSchema>;
+
+export function HederaForm({ onSubmit }: HederaFormProps) {
+  const formik = useFormik<HederaFormValues>({
+    onSubmit,
+    initialValues: {
+      serializedSignedTx: '',
+      startTime: undefined,
```

Comment:
nit: `startTime: null`

Reply:
expected values from the validation schema are are number or undefined